stdpopsim/slim_engine.py

```diff
@@ -1833,6 +1839,41 @@ def _get_msp_rate_map(breaks, is_this_dfe, rate):
                 )
             )
 
+        # If haploid, rebuild individual table so each sample is an individual
```

Comment:
This *looks* right, but a less error-prone way to do it would involve lots of calls to `tables.add_row( )`; did you do it this way because it's faster?

Reply:
agreed, and done